### PR TITLE
[#3645] Airy deployment on gke using terraform

### PR DESCRIPTION
**Pull request by bilge-cakmak**
Currently tested and operational on GCP. 

[note-1] In order to accommodate clusters using the same network, the "gcp-gke" module needs to be modified.
[note-2] The kube.conf file generated by the "gke auth" module has credentials that are only valid for one hour.

### CODE REVIEW

File: infrastructure/terraform/install/gcp-gke/main.tf [hunk 1]
```diff
@@ -1,6 +1,10 @@
 module "gcp-gke" {
-  source = "github.com/airyhq/airy.git/infrastructure/terraform/modules/gcp-gke"
+  #source = "github.com/airyhq/airy.git/infrastructure/terraform/modules/gcp-gke"
+  source = "../../modules/gcp-gke"
```
Comment (ljupcovangelski) [inline infrastructure/terraform/install/gcp-gke/main.tf:3]: This we need to revert before merge.

**Review by ljupcovangelski — COMMENTED** [review]
Tested it out and it works great :tada:

File: infrastructure/terraform/install/gcp-gke/main.tf (hunk 1 above)
Comment (bilge-cakmak) [reply infrastructure/terraform/install/gcp-gke/main.tf:3]: 👍

**Review by ljupcovangelski — APPROVED** [review]
Nice :+1:

**Pull request merged**